documentation/source/runner.rst

```diff
@@ -0,0 +1,47 @@
+.. _howto-runner:
+
+******************************
+Building HDL and Running Tests
+******************************
+
+.. versionadded:: 1.7
+
+The Python Test Runner described here is a replacement
+for cocotb's traditional Makefile flow.
+It builds the HDL for the simulator selected by
+setting :make:var:`SIM` and runs the tests.
```

Comment:
I had missed the fact that it's not the runner which evaluates `SIM` but the test file.
Probably something to discuss in the call.